compiler/src/dotty/tools/dotc/transform/ElimRepeated.scala

```diff
@@ -85,7 +102,8 @@ class ElimRepeated extends MiniPhase with InfoTransformer { thisPhase =>
   private def isVarargsMethod(sym: Symbol)(using Context) =
     hasVarargsAnnotation(sym) ||
       hasRepeatedParams(sym) &&
-      (sym.allOverriddenSymbols.exists(s => s.is(JavaDefined) || hasVarargsAnnotation(s)))
+        overridesJava(sym)
```

Comment:
The previous version skipped the checks on overriden symbols when there were no repeated params (hence the parens). Is it intentional to change that?

Reply:
No, this is missing a pair of parens